Gulpfile.js

```diff
@@ -157,9 +157,9 @@ gulp.task('html-hint', () => {
 
 // Test task
 gulp.task('test', ['check-handlebars', 'standard', 'html-hint'], () => {
-  env.set({
-    PORT: 8001
-  })
+  const port = parseInt(process.env.WASTE_PERMITS_APP_PORT) + 1
```

Comment:
It's no longer called `WASTE_PERMITS_APP_PORT`, simply `PORT`. This change was made in PR #32 to support running on Heroku.